examples/default/example.tf

```diff
@@ -0,0 +1,92 @@
+# ----------------------------------------
+# Create a service fargate
+# ----------------------------------------
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+resource "aws_ecs_cluster" "fargate_cluster" {
```

Comment:
In my opinion, it's implied that these resources are for fargate since it is a standalone fargte module. So I think you should just cut `fargate_` from `fargate_cluster` and make it just `cluster`-  same goes for other resource/module names in this file 👍